plugins/woocommerce/client/blocks/bin/webpack-configs.js

```diff
@@ -129,15 +129,6 @@ const getCoreConfig = ( options = {} ) => {
 				bundleAnalyzerReportTitle: 'Core',
 			} ),
 			new ProgressBarPlugin( getProgressBarPluginConfig( 'Core' ) ),
-			new CreateFileWebpack( {
```

Comment:
Love it! Can we also drop the dependency `create-file-webpack` and it's import in this file please?

Reply:
Whoops, thanks for spotting. Checked blocks and it's not used anywhere else. removed in https://github.com/woocommerce/woocommerce/pull/56515/commits/5f1dcee0e69c90bd39650e649db6366c8ebe63d4